pkg/secrets/fetch_secret.go

```diff
@@ -150,15 +150,20 @@ func fetchSecret(secretsHandle []string, origin string) (map[string]string, erro
 		if v.ErrorMsg != "" {
 			return nil, fmt.Errorf("an error occurred while decrypting '%s': %s", sec, v.ErrorMsg)
 		}
-		if v.Value == "" {
+
+		value := strings.TrimSpace(v.Value)
```

Comment:
Should we trim whitespaces or only line break at the start and end of the secret ?

Reply:
I think both are probably acceptable; we might get some support cases, but I think it'll be a rare occurrence. We'll have to advertise this in the release notes.